packages/destination-actions/src/destinations/accoil-analytics/group/index.ts

```diff
@@ -0,0 +1,80 @@
+import type { ActionDefinition } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+import { commonFields } from '../common-fields'
+import { endpointUrl } from '../utils'
+
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Group',
+  description: 'Identify Accounts (groups) in Accoil',
+  defaultSubscription: 'type = "identify"',
```

Comment:
should this be group?
```suggestion
  defaultSubscription: 'type = "group"',
```